CHANGELOG.md

```diff
@@ -34,8 +34,11 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   The default SCC for sensor is now `restricted[-v2]` or `stackrox-sensor` depending on the settings.
   Both the `runAsUser` and `fsGroup` for the admission-control and sensor deployments are no longer hardcoded to 4000 on Openshift clusters
   to allow using the `restricted` and `restricted-v2` SCCs.
-- The service account "central", which is used by the central deployment, will now include `get and list` access to the pod resource in the namespace
-  where central is deployed to. This fixes an issue when generating diagnostic bundles to now correctly include all logs within the namespace of central.
+- The service account "central", which is used by the central deployment, will now include `get and list` access to the following resources in the namespace where central is deployed to:
+  - "pods"
+  - "events"
+  - "namespaces"
```

Comment:
I would write this as an inline list, and also use backticks instead of double quotes:
```suggestion
`pods`, `events`, and `namespaces`
```